lib/libutee/include/pta_attestation.h

```diff
@@ -0,0 +1,99 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (C) 2021, Huawei Technologies Co., Ltd
+ */
+
+/*
+ * Provide remote attestation services
+ */
+
+#ifndef __PTA_ATTESTATION_H
+#define __PTA_ATTESTATION_H
+
+#define PTA_ATTESTATION_UUID { 0x39800861, 0x182a, 0x4720, \
+		{ 0x9b, 0x67, 0x2b, 0xcd, 0x62, 0x2b, 0xc0, 0xb5 } }
+
+/*
+ * Get the RSA public key use for signing the values returned by other commands.
+ *
+ * [out]    memref[0]        Public key exponent in big endian order
+ * [out]    memref[1]        Modulus in big endian order
+ *
+ * Return codes:
+ * TEE_SUCCESS
+ * TEE_ERROR_GENERIC - Internal error
+ * TEE_ERROR_SHORT_BUFFER - One or both buffers are too small, required size
+ *                          is provided in memref[i].size
+ */
+#define PTA_ATTESTATION_GET_PUBKEY 0x0
```

Comment:
Although currently we are using RSA as the signing scheme but it should be possible in future to extend this pTA to use ECDSA scheme instead in a backwards compatible way. So I would suggest you to add an output value parameter to say that is an RSA public key.

Reply:
I agree that other encryption schemes may be useful in the future, but I'd rather avoid introducing too much complexity now. I think what you are asking can easily be done with minimal code changes by adding a new command.

Reply:
Yeah adding a new info command would be good alternative. I think it would be worth to add at this point. As it would avoid to hard code signature type and algorithm used in both pseudo TA and remote client when that info can be fetched at runtime.

Reply:
How about `PTA_ATTESTATION_GET_PUBKEY` would return `TEE_ALG_RSASSA_PKCS1_PSS_MGF1_SHA256` as `[out] value[2].a`?

Reply:
That's fine with me as well.